go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	bou.ke/monkey v1.0.2
 	github.com/gorilla/mux v1.7.4
-	github.com/mattermost/mattermost-server/v5 v5.24.1
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200717105608-93a537a63629
```

Comment:
What changes in `mattermost-server` that are only in `master` are needed for this PR?

Reply:
getting the `channel_id` in the dynamic argument endpoint would be the one.

Reply:
Got it :+1: